packages/components/Molecules/Form/fields/fieldRploptionbutton.vue

```diff
@@ -42,6 +42,7 @@ export default {
   $rpl-option-button-label-color: rpl-color('extra_dark_neutral') !default;
   $rpl-option-button-label-border-width: 1px !default;
   $rpl-option-button-label-border: $rpl-option-button-label-border-width solid rpl-color('mid_neutral_1') !default;
+  $rpl-option-button-label-border-radius: 4px;
```

Comment:
Hey @dylankelly I think you can reuse global setting `$rpl-form-element-border-radius` so all form element border radius are in same style. https://github.com/dpc-sdp/ripple/blob/3358427e782379eae15f500b0a8aaa34a30169f1/packages/components/Molecules/Form/scss/_form.scss#L10

Reply:
✅